instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/db/SqlStatementSanitizer.java

```diff
@@ -28,7 +27,7 @@ public static SqlStatementInfo sanitize(String statement) {
     return sqlToStatementInfoCache.computeIfAbsent(
         statement,
         k -> {
-          log.trace("SQL statement cache miss");
+          supportability.incrementCounter("SqlStatementSanitizer cache miss");
```

Comment:
It would be nice to have these counter names be constants in the SupportabilityMetrics class, so it's easy to see what the various counters are. It won't stop someone from adding custom counter names, but it will at least add a little bit to the self-documenting nature of the code.